### PR TITLE
Feat/issue 43 update table component

### PR DESCRIPTION
**Pull request by Vojimirovich**

### CODE REVIEW

File: frontend/apps/web/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "start": "http-server dist -P http://localhost:8080? -g",
     "preserve": "yarn prebuild",
     "serve": "cross-env NODE_ENV=development webpack serve --config config/webpack.dev.js",
-    "prebuild": "concurrently \"yarn generate:routes\" \"yarn generate:behaviors\"",
```
Comment (aleksaToljic) [inline frontend/apps/web/package.json:13]: This shouldn't be in commit

File: frontend/packages/ui/src/Table.tsx [hunk 2]
```diff
@@ -116,34 +126,33 @@ export const Table = ({
...
-                  </Column>
-                ) : index === 0 && isLoading ? (
-                  <Column width={'100%'} isFirst isLast key={index}>
-                    <TableHeaderText> </TableHeaderText>
-                  </Column>
-                ) : null
-              })}
-          </tr>
-        </THead>
+        <TableHeaderRow>
```
Comment (aleksaToljic) [inline frontend/packages/ui/src/Table.tsx:129]: You lost th here, we should return it.

**Review by aleksaToljic — CHANGES_REQUESTED** [review]

File: frontend/apps/web/package.json (hunk 1 above)
Comment (Vojimirovich) [reply frontend/apps/web/package.json]: Done

File: frontend/packages/ui/src/Table.tsx (hunk 2 above)
Comment (Vojimirovich) [reply frontend/packages/ui/src/Table.tsx]: Returned it, thanks for noticing!

**Review by aleksaToljic — APPROVED** [review]

**Pull request merged**